bin/run

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env node
 
-require('@oclif/command').run()
-.then(require('@oclif/command/flush'))
-.catch(require('@oclif/errors/handle'))
+const oclif_command = require("@oclif/command");
+const oclif_cmd_flush = require("@oclif/command/flush");
+const oclif_err_handle = require("@oclif/errors/handle");
+
+oclif_command.run().then(oclif_cmd_flush).catch(oclif_err_handle);
```

Comment:
Why is this necessary?

Reply:
totally not necessary; but since there were problems when people were installing from their local clone, I figured this may give them a better error (i.e., `oclif` failed to import properly rather than `oclif` failed to run properly)

can revert if you'd like.

Reply:
I see, let us just keep it then 👍